Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/pistol.yml

```diff
@@ -41,3 +41,15 @@
     - CMCartridgeRevolver44
   - type: CartridgeAmmo
     proto: CMBulletRevolver44
+
+- type: entity
+  id: CMCartridgeRevolver44Marksman
+  name: cartridge (.44 Marksman)
+  parent: CMCartridgePistolBase
+  components:
+  - type: Tag
+    tags:
+    - Cartridge
+    - CMCartridgeRevolver44
+  - type: CartridgeAmmo
+    proto: CMBulletRevolver44Marksman
```

Comment:
```suggestion
    proto: CMBulletRevolver44Marksman
    deleteOnSpawn: true
```

Reply:
This won't affect the cartridges spawning but is probably needed for the system change